domain/application/state/unit.go

```diff
@@ -2077,7 +2077,7 @@ ON CONFLICT (application_uuid) DO UPDATE SET
 		return errors.Capture(err)
 	}
 
-	appID, err := st.getUnitApplicationID(ctx, tx, unitUUID)
+	appID, err := st.getUnitApplicationUUID(ctx, tx, unitUUID)
 	if err != nil {
 		return errors.Errorf("getting application id for unit %q: %w", unitUUID, err)
```

Comment:
There might be other cases of this, can we please fix up the error strings and possibly comments as well. Otherwise I'd rather not take on the cost of the churn this will cause so close to release.

Reply:
Yeah of course. Done it. 
The changeset is now huge, but I think we should be fine (hope no error slipped over).